### PR TITLE
Cart

### PR DESCRIPTION
**Pull request by chanchal16**
- created cart page.
- 
Screenshot (260)

### CODE REVIEW

**Review by ravikumar0403 — APPROVED** [review]
Looks good to me

File: features/cart.html [hunk 1]
```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel='stylesheet' href='https://breezecss.netlify.app/styles/breeze.css' >
```
Comment (bharati-21) [inline features/cart.html:7]: Suggestion - Add author and desc meta tag

**Review by bharati-21 — APPROVED** [review]
Looks good to me!
Nice touch by adding comments in CSS :)

File: features/cart.html (hunk 1 above)
Comment (chanchal16) [reply features/cart.html:7]: okay, will add that. Thank you!!

**Pull request merged**